### PR TITLE
feat: support opbnb info page

### PR DESCRIPTION
**Pull request by Chef-Yogi**
### Summary
🌐📊🔗

This pull request adds support for the OPBNB chain, a new Optimistic Rollup network, to the info and V3 info pages of the web app. It defines the subgraph endpoint, block explorer, and chain ID constants for the OPBNB chain, and uses them in the data fetching and display logic. It also updates the UI components to include the OPBNB chain in the network switcher and the navigation bar.

> _We're breaking the chains of the old world order_
> _We're riding the wave of the OPBNB roller_
> _We're fetching the data from the subgraph explorer_
> _We're building the future on the Optimistic border_

### Walkthrough
*  Add support for OPBNB chain on info pages ([link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-7238357db46da6bb48ca41841457ac0e832ee21fb8fd76a6b3c9dbf701ce8c62R35), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-7238357db46da6bb48ca41841457ac0e832ee21fb8fd76a6b3c9dbf701ce8c62L69-R70), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80R7), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80L20-R24), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80R36), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80R51), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80R64), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80R75), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80R86), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80R97), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80R108), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80R119), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80R131), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80R145), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-f2387797bca27b530d156fa7f58d5addb1dc1d2ff995c1bb9b5dd10be464b292R311-R312), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-ae2fef01f2e4cbfcfafd8be2bd435854da0b178fbbbd7846f8a68194141de72aL25-R25), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-ae2fef01f2e4cbfcfafd8be2bd435854da0b178fbbbd7846f8a68194141de72aL98-R98), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-ae2fef01f2e4cbfcfafd8be2bd435854da0b178fbbbd7846f8a68194141de72aL104-R104), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-bb26c7b8dd2db305bc739e7e4f031774af7349a6d30a7bf6431d335abf2a5c0eL22-R22), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-bb26c7b8dd2db305bc739e7e4f031774af7349a6d30a7bf6431d335abf2a5c0eL77-R77), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-bb26c7b8dd2db305bc739e7e4f031774af7349a6d30a7bf6431d335abf2a5c0eL83-R83), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-77f9b222e9f5de52041b06ae540e3a8cb32c1fd777c20be054eb88990cabc845R65))
  * Add `BLOCKS_CLIENT_OPBNB` constant to `endpoints.ts` for OPBNB subgraph URL ([link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-7238357db46da6bb48ca41841457ac0e832ee21fb8fd76a6b3c9dbf701ce8c62R35))
  * Use `BLOCKS_CLIENT_OPBNB` constant in `BLOCKS_CLIENT_WITH_CHAIN` object ([link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-7238357db46da6bb48ca41841457ac0e832ee21fb8fd76a6b3c9dbf701ce8c62L69-R70))
  * Import `BLOCKS_CLIENT_OPBNB` constant and `opBNB` chain from `wagmi/chains` in `constant.ts` ([link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80R7), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80L20-R24))
  * Add mappings for OPBNB chain in `multiChainName`, `multiChainQueryMainToken`, `multiChainBlocksClient`, `multiChainStartTime`, `multiChainId`, `multiChainPaths`, `multiChainQueryClient`, `multiChainScan`, `multiChainWhitelist`, and `multiChainBlacklist` objects ([link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80R36), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80R51), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80R64), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80R75), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80R86), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80R97), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80R108), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80R119), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80R131), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-85dec4f2520ab4d031442a39680ae804a521ef025a33f9b42fb151b108015d80R145))
  * Add case for `'opbnb'` in `useChainNameByQuery` hook in `hooks.ts` ([link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-f2387797bca27b530d156fa7f58d5addb1dc1d2ff995c1bb9b5dd10be464b292R311-R312))
  * Import `opBNB` chain from `wagmi/chains` in `InfoNav` components for V2 and V3 info pages in `InfoNav/index.tsx` and `Layout/InfoNav.tsx` ([link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-ae2fef01f2e4cbfcfafd8be2bd435854da0b178fbbbd7846f8a68194141de72aL25-R25), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-bb26c7b8dd2db305bc739e7e4f031774af7349a6d30a7bf6431d335abf2a5c0eL22-R22))
  * Add `opBNB` chain to `targetChains` array in `NetworkSwitcher` components for V2 and V3 info pages ([link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-ae2fef01f2e4cbfcfafd8be2bd435854da0b178fbbbd7846f8a68194141de72aL98-R98), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-bb26c7b8dd2db305bc739e7e4f031774af7349a6d30a7bf6431d335abf2a5c0eL77-R77))
  * Add fallback value of `0` for `foundChain?.id` expression in `NetworkSwitcher` components for V2 and V3 info pages ([link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-ae2fef01f2e4cbfcfafd8be2bd435854da0b178fbbbd7846f8a68194141de72aL104-R104), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-bb26c7b8dd2db305bc739e7e4f031774af7349a6d30a7bf6431d335abf2a5c0eL83-R83))
  * Add mapping for OPBNB chain in `SUBGRAPH_START_BLOCK` object in `constants.ts` for V3 info pages ([link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-77f9b222e9f5de52041b06ae540e3a8cb32c1fd777c20be054eb88990cabc845R65))
* Add fallback values for `blocks` and `address` parameters in hooks for fetching pool and token data in `hooks.ts` ([link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-f2387797bca27b530d156fa7f58d5addb1dc1d2ff995c1bb9b5dd10be464b292L86-R86), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-f2387797bca27b530d156fa7f58d5addb1dc1d2ff995c1bb9b5dd10be464b292L94-R94), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-f2387797bca27b530d156fa7f58d5addb1dc1d2ff995c1bb9b5dd10be464b292L102-R102), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-f2387797bca27b530d156fa7f58d5addb1dc1d2ff995c1bb9b5dd10be464b292L147-R147), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-f2387797bca27b530d156fa7f58d5addb1dc1d2ff995c1bb9b5dd10be464b292L174-R174), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-f2387797bca27b530d156fa7f58d5addb1dc1d2ff995c1bb9b5dd10be464b292L199-R199), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-f2387797bca27b530d156fa7f58d5addb1dc1d2ff995c1bb9b5dd10be464b292L223-R223))
  * Add fallback value of `[]` for `blocks` parameter in `useAllPoolDataSWR`, `usePoolDatasSWR`, `useAllTokenHighLight`, and `useAllTokenDataSWR` hooks ([link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-f2387797bca27b530d156fa7f58d5addb1dc1d2ff995c1bb9b5dd10be464b292L86-R86), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-f2387797bca27b530d156fa7f58d5addb1dc1d2ff995c1bb9b5dd10be464b292L102-R102), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-f2387797bca27b530d156fa7f58d5addb1dc1d2ff995c1bb9b5dd10be464b292L147-R147), [link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-f2387797bca27b530d156fa7f58d5addb1dc1d2ff995c1bb9b5dd10be464b292L174-R174))
  * Change return type of `usePoolDatasSWR` hook from `PoolData[]` to `(PoolData | undefined)[]` ([link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-f2387797bca27b530d156fa7f58d5addb1dc1d2ff995c1bb9b5dd10be464b292L94-R94))
  * Add type annotation of `Array<string[]>` for `addressGroup` variable in `fetcher` function ([link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-f2387797bca27b530d156fa7f58d5addb1dc1d2ff995c1bb9b5dd10be464b292L184-R184))
  * Add fallback value of `[]` for `blocks` parameter in `useTokenDatasSWR` hook ([link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-f2387797bca27b530d156fa7f58d5addb1dc1d2ff995c1bb9b5dd10be464b292L199-R199))
  * Add fallback value of `''` for `address` parameter in `useTokenDataSWR` hook ([link](https://github.com/pancakeswap/pancake-frontend/pull/8100/files?diff=unified&w=0#diff-f2387797bca27b530d156fa7f58d5addb1dc1d2ff995c1bb9b5dd10be464b292L223-R223))

### CODE REVIEW

File: apps/web/src/views/Info/components/InfoNav/index.tsx [hunk 1]
```diff
@@ -95,13 +95,13 @@ const InfoNav: React.FC<{ isStableSwap: boolean }> = ({ isStableSwap }) => {
...
 
-const targetChains = [mainnet, bsc, polygonZkEvm, zkSync, arbitrum, linea, base]
+const targetChains = [mainnet, bsc, polygonZkEvm, zkSync, arbitrum, linea, base, opBNB]
 
 export const NetworkSwitcher: React.FC<{ activeIndex: number }> = ({ activeIndex }) => {
   const { t } = useTranslation()
   const chainName = useChainNameByQuery()
   const foundChain = chains.find((d) => d.id === multiChainId[chainName])
-  const symbol = multiChainShortName[foundChain?.id] ?? foundChain?.nativeCurrency?.symbol
+  const symbol = multiChainShortName[foundChain?.id ?? 0] ?? foundChain?.nativeCurrency?.symbol
```
Comment (ChefJerry) [inline apps/web/src/views/Info/components/InfoNav/index.tsx:104]: use `0` seems different with preview logic

Comment (ChefMomota) [reply apps/web/src/views/Info/components/InfoNav/index.tsx:104]: How about make it.

`const symbol = foundChain?.id && multiChainShortName[foundChain?.id] ?  multiChainShortName[foundChain.id] : foundChain?.nativeCurrency?.symbol`

Comment (ChefJerry) [reply apps/web/src/views/Info/components/InfoNav/index.tsx:104]: yeah, that will be right

Comment (Chef-Yogi) [reply apps/web/src/views/Info/components/InfoNav/index.tsx:104]: update to -1, this is for fixing ts check

**Review by ChefJerry — APPROVED** [review]

**Review by chefjackson — APPROVED** [review]

**Pull request merged**